src/main/resources/templates/bars/show.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html xmlns:th="https://www.thymeleaf.org"
```

Comment:
use whitespace to your advantage to show the hierarchy of html elements visually. Please add indentation here and in your other HTML files.